lib/UI/UI.rb

```diff
@@ -40,7 +40,7 @@ def show_grid board
         @out.print DIVIDER if (row < size)
         @out.print PLUS if (col < size and row < size)
       end
-      @out.print "\n"
+      @out.print "\n"if (row < size)
```

Comment:
Should be a space between the if and what comes before. Also don't need the parenthesis